src/com/goide/runconfig/GoRunConfigurationBase.java

```diff
@@ -194,4 +195,14 @@ public String getWorkingDirectory() {
   public void setWorkingDirectory(@NotNull String workingDirectory) {
     myWorkingDirectory = workingDirectory;
   }
+
+  @NotNull
+  public String getFilePath() {
```

Comment:
Why do you need it in base class? Why do you need duplicates of this methods in GoTestRunConfiguration?